chain/icon/sender.go

```diff
@@ -64,14 +66,32 @@ type sender struct {
 }
 
 func (s *sender) newTransactionParam(prev string, rm *RelayMessage) (*TransactionParam, error) {
-	b, err := codec.RLP.MarshalToBytes(rm)
+	for i, bur := range rm.BlockUpdates {
+		s.l.Debugf("msg.BlockUpdates[%d]: %x\n", i, bur)
+	}
+
+	prm := &struct {
+		BlockUpdates [][]byte
+		BlockProof   []byte
+		StateProof   *pra.StateProof
+	}{
+		BlockUpdates: rm.BlockUpdates,
+		BlockProof:   rm.BlockProof,
+		StateProof:   nil,
+	}
+
+	b, err := rlp.EncodeToBytes(prm)
```

Comment:
@trmaphi What is the bug here? 
I haven't got any error when using the codec package in BTP to send RM from ICON to PARACHAIN. 
Please point out what is the bug. Because it might affect to the smart-contract team as well.